swagger/v1/api_v1.json

```diff
@@ -9,8 +9,12 @@
     "/api/articles": {
       "get": {
         "summary": "Published articles",
-        "security": [],
-        "tags": ["articles"],
+        "security": [
+
+        ],
```

Comment:
```suggestion
        "security": [],
```

Reply:
Thanks for this suggestion @Ridhwana! It's not recommended to edit this JSON file manually because it's autogenerated by RSwag so it will continue to pop up in the future though 🙂

Reply:
Ah I missed that! 😁 Thanks @fdocr.